frontend/src/data_context_global.tsx

```diff
@@ -6,22 +6,35 @@ export interface NameID {
     id: number
 }
 
+export interface HerdNameID {
+    herd_name: string,
+    herd: string,
+    id: number
+}
+
 export interface DateValue {
     date: string
     value: string | number
     id?: number
 }
 
+export interface LimitedIndividual {
+    id: number,
+    name: string | null,
+    number: string,
+    sex?: string,
+}
+
 export interface Individual {
```

Comment:
You can do `export interface Individual extends LimitedIndividual` to avoid repeating fields (however seems like the type of `sex` is slightly incompatible)

Reply:
good point! Once all the types are in place we should really go through them and see what we can smash together...